brd-ios/breadwallet/src/Views/PinView.swift

```diff
@@ -18,7 +18,7 @@ class PinView: UIView {
 
     // MARK: - Public
     var itemSize: CGFloat {
-        return 24.0
+        return 22.0
```

Comment:
why?

Reply:
smaller pin dots
<img width="300" alt="Screenshot 2022-04-21 at 13 15 37" src="https://user-images.githubusercontent.com/40755142/164446682-7167ab11-6b21-4fa1-8886-6b7e1f9ee7ff.png">

Reply:
I hope this doesn't break anything as the pin pad is being used in many other places.